tests/python/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright Contributors to the OpenColorIO Project.
 
+include(FindPythonPackage)
+
+# NumPy
+# https://pypi.org/project/numpy/
+find_python_package(numpy REQUIRED)
```

Comment:
Are we sure we really want to make this a requirement?  Could it be optional?

Reply:
Good point. I was on the fence about it. I have now made it optional and include warnings when running Python tests that test coverage is incomplete without it. The `CPUProcessor` tests will still run, but only the two apply methods that accept a Python list.